notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb

```diff
@@ -1157,16 +1067,9 @@
         "id": "make_batch_predictions:migration"
       },
       "source": [
-        "## Make batch predictions"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "batchpredictionjobs_create:migration,new,mbsdk"
-      },
-      "source": [
-        "### [predictions.batch-prediction](https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions)"
+        "### Generate batch predictions\n",
+        "\n",
+        "Here is the [Overview of getting predictions on Vertex AI](https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions)"
```

Comment:
This URL involves a redirect. 
- Update from: https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions
- To:  https://cloud.google.com/vertex-ai/docs/predictions/overview